OTKit/otkit-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "otkit-icons",
-  "version": "7.13.0",
+  "version": "8.13.0",
```

Comment:
If you are bumping to a major version, shouldn't it be `8.0.0` ?

Reply:
@ronniechong Good catch. That was a major brain fart. Thanks! Fixed